swagger/src/test/scala/org/http4s/rho/swagger/Arbitraries.scala

```diff
@@ -62,11 +62,11 @@ object Arbitraries {
 
   def genMediaType: Gen[String] =
     oneOf(
-      `application/json`,
-      `audio/midi`,
-      `image/gif`,
-      `text/html`
-    ).map(_.renderString)
+      application.json,
+      audio.midi,
+      image.gif,
+      text.html
+    ).map(_.toString)
```

Comment:
Again, not sure `toString` should be the substitute for `renderString` on a `MediaRange`

Reply:
It seems that we have `Show` instance for `MediaType`. I believe it will be the correct substitution.

Reply:
I believe you are correct.